nvme-print.c

```diff
@@ -281,19 +281,19 @@ static void json_nvme_id_ctrl(struct nvme_id_ctrl *ctrl,
 	json_object_add_value_int(root, "hctma", le16_to_cpu(ctrl->hctma));
 	json_object_add_value_int(root, "mntmt", le16_to_cpu(ctrl->mntmt));
 	json_object_add_value_int(root, "mxtmt", le16_to_cpu(ctrl->mxtmt));
-	json_object_add_value_int(root, "sanicap", le32_to_cpu(ctrl->sanicap));
-	json_object_add_value_int(root, "hmminds", le32_to_cpu(ctrl->hmminds));
+	json_object_add_value_uint(root, "sanicap", le32_to_cpu(ctrl->sanicap));
+	json_object_add_value_uint(root, "hmminds", le32_to_cpu(ctrl->hmminds));
```

Comment:
also for `hmminds` we should update the `%d` format specifier.

Could you double check that all the print statements for JSON and terminal output agree on the output format? Thanks!

Reply:
Just pushed the additional patch to update the mcl and hmminds normal terminal output print format.
Also just checked the outputs as [test.txt](https://github.com/linux-nvme/nvme-cli/files/10260841/test.txt) but sorry some command outputs are not supported on my environment so not tested actually.